streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfo.java

```diff
@@ -128,6 +128,7 @@ public static AssignmentInfo decode(ByteBuffer data) {
             if (version < 0 || version > CURRENT_VERSION) {
                 TaskAssignmentException ex = new TaskAssignmentException("Unknown assignment data version: " + version);
                 log.error(ex.getMessage(), ex);
+                in.close();
```

Comment:
Yes, actually I'll try to use try with resources here in the next update. Thanks for catching it.